paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -298,19 +298,21 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
         when (clientSecret) {
             is PaymentIntentClientSecret -> {
                 collectBankAccountLauncher?.presentWithPaymentIntent(
-                    lazyPaymentConfig.get().publishableKey,
-                    clientSecret.value,
-                    CollectBankAccountConfiguration.USBankAccount(
+                    publishableKey = lazyPaymentConfig.get().publishableKey,
+                    stripeAccountId = lazyPaymentConfig.get().stripeAccountId,
```

Comment:
@jameswoo-stripe Looks like PaymentSheet would be having this this issue as well, fixed that too.

Reply:
Yea, thanks for getting this fix in!